### ISSUE TITLE
Most features are basically broken randomly on vscode 1.43

### ISSUE THREAD

**Issue by AlseinX** [dotnet/vscode-csharp#3694]
## Environment data
`dotnet --info` output:
```
.NET Core SDK (reflecting any global.json):
 Version:   3.1.201
 Commit:    b1768b4ae7

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.19041
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\3.1.201\

Host (useful for support):
  Version: 3.1.3
  Commit:  4a9f85e9f8

.NET Core SDKs installed:
  3.1.201 [C:\Program Files\dotnet\sdk]

.NET Core runtimes installed:
  Microsoft.AspNetCore.App 3.1.3 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]        
  Microsoft.NETCore.App 3.1.3 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.WindowsDesktop.App 3.1.3 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]

To install additional .NET Core runtimes or SDKs:
  https://aka.ms/dotnet-download
```
```
.NET Core SDK (reflecting any global.json):
 Version:   3.1.201
 Commit:    b1768b4ae7

Runtime Environment:
 OS Name:     debian
 OS Version:  10
 OS Platform: Linux
 RID:         debian.10-x64
 Base Path:   /usr/share/dotnet/sdk/3.1.201/

Host (useful for support):
  Version: 3.1.3
  Commit:  4a9f85e9f8

.NET Core SDKs installed:
  3.1.201 [/usr/share/dotnet/sdk]

.NET Core runtimes installed:
  Microsoft.AspNetCore.App 3.1.3 [/usr/share/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 3.1.3 [/usr/share/dotnet/shared/Microsoft.NETCore.App]

To install additional .NET Core runtimes or SDKs:
  https://aka.ms/dotnet-download
```
VS Code version: `1.43.2`
C# Extension version: `1.21.15`

## Steps to reproduce
Open and edit a fresh new `dotnet new classlib` project.

## Expected  behavior
Everything would work fine, and if there is a problem, there would be some warn or fail output in the `OmniSharp Log`

## Actual behavior
Language support would never start until you make a change to the source file.
No warn or fail reported, and everything looks harmonious.
Auto-completion for adding namespace using disappears.
All compiler errors are not detected except for syntax lexer/parser errors.
Types from nuget packages are not respected.
Some type inference does not work.
Phantom break points when debugging.
Build & launch sometimes have different behavior from the output built from command line, which bring strange problems like `.so` file not found when running a asp.net core project on linux, some properties in `.props` are ignored, or some project does not output executable files.

Random collections of these problems appear by different, and are even not able to determine whether they appear randomly or consistently. The occurrence of these problems could vary from different OS and/or project types.

I tried on both on Windows and WSL2 Debian, both default or `"latest"` version.
There are always some of these problems that make VSCode current version not an usable C# editor any more.

**Comment by JoeRobich** [comment]
@AlseinX Sorry you are experiencing these issues. Can you please share your OmniSharp log and if possible a repo that repros this behavior?

**Comment by atrauzzi** [comment]
I'm noticing on `1.43.2` that omnisharp just isn't doing _anything_, or appears to be having a very hard time in my projects.

It's hard to explain because the only symptoms I'm seeing are behavioural.  It manifests in weird ways like intellisense not working _at all_, ctrl-clicking on anything results in the little progress bar running across the top of the screen, stopping and then nothing happening.  I can't refactor anything.

It just seems....dead?

Logs show nothing out of the ordinary (I think?)...

```
Starting OmniSharp server at 2020-04-04, 10:41:14 a.m.
    Target: /my/project/path/MyProject

OmniSharp server started.
    Path: /home/atrauzzi/.vscode/extensions/ms-dotnettools.csharp-1.21.16/.omnisharp/1.34.15/run
    PID: 25382

[info]: OmniSharp.Stdio.Host
        Starting OmniSharp on ubuntu 19.10 (x64)
[info]: OmniSharp.Services.DotNetCliService
        DotNetPath set to dotnet
[info]: OmniSharp.MSBuild.Discovery.MSBuildLocator
        Located 1 MSBuild instance(s)
            1: StandAlone 16.4 - "/home/atrauzzi/.vscode/extensions/ms-dotnettools.csharp-1.21.16/.omnisharp/1.34.15/omnisharp/.msbuild/Current/Bin"
[info]: OmniSharp.MSBuild.Discovery.MSBuildLocator
        MSBUILD_EXE_PATH environment variable set to '/home/atrauzzi/.vscode/extensions/ms-dotnettools.csharp-1.21.16/.omnisharp/1.34.15/omnisharp/.msbuild/Current/Bin/MSBuild.dll'
[info]: OmniSharp.MSBuild.Discovery.MSBuildLocator
        Registered MSBuild instance: StandAlone 16.4 - "/home/atrauzzi/.vscode/extensions/ms-dotnettools.csharp-1.21.16/.omnisharp/1.34.15/omnisharp/.msbuild/Current/Bin"
            CscToolExe = csc.exe
            MSBuildToolsPath = /home/atrauzzi/.vscode/extensions/ms-dotnettools.csharp-1.21.16/.omnisharp/1.34.15/omnisharp/.msbuild/Current/Bin
            CscToolPath = /home/atrauzzi/.vscode/extensions/ms-dotnettools.csharp-1.21.16/.omnisharp/1.34.15/omnisharp/.msbuild/Current/Bin/Roslyn
            BypassFrameworkInstallChecks = true
            MSBuildExtensionsPath = /home/atrauzzi/.vscode/extensions/ms-dotnettools.csharp-1.21.16/.omnisharp/1.34.15/omnisharp/.msbuild
[info]: OmniSharp.Cake.CakeProjectSystem
        Detecting Cake files in '/my/project/path/MyProject'.
[info]: OmniSharp.Cake.CakeProjectSystem
        Could not find any Cake files
[info]: OmniSharp.MSBuild.ProjectSystem
        No solution files found in '/my/project/path/MyProject'
[info]: OmniSharp.MSBuild.ProjectManager
        Queue project update for '/my/project/path/MyProject/MyProject.csproj'
[info]: OmniSharp.Script.ScriptProjectSystem
        Detecting CSX files in '/my/project/path/MyProject'.
[info]: OmniSharp.Script.ScriptProjectSystem
        Found 1 CSX files.
[info]: OmniSharp.Script.ScriptContextProvider
        Searching for compilation dependencies with the fallback framework of 'net461'.
[info]: OmniSharp.MSBuild.ProjectManager
        Loading project: /my/project/path/MyProject/MyProject.csproj
[info]: OmniSharp.Script.ScriptContextProvider
        Unable to find dependency context for CSX files. Will default to non-context usage (Desktop CLR scripts).
[info]: OmniSharp.MSBuild.ProjectManager
        Successfully loaded project file '/my/project/path/MyProject/MyProject.csproj'.
[info]: OmniSharp.MSBuild.ProjectManager
        Adding project '/my/project/path/MyProject/MyProject.csproj'
[fail]: OmniSharp.MSBuild.ProjectManager
        Failed to add project to workspace: '/my/project/path/MyProject/MyProject.csproj'
[fail]: OmniSharp.MSBuild.ProjectManager
        Could not locate project in workspace: /my/project/path/MyProject/MyProject.csproj
[info]: OmniSharp.Script.ScriptProjectSystem
        Added CSX project '/my/project/path/MyProject/csx/mucking.csx' to the workspace.
[info]: OmniSharp.WorkspaceInitializer
        Invoking Workspace Options Provider: OmniSharp.Roslyn.CSharp.Services.CSharpFormattingWorkspaceOptionsProvider, Order: 0
[info]: OmniSharp.WorkspaceInitializer
        Invoking Workspace Options Provider: OmniSharp.Roslyn.CSharp.Services.RenameWorkspaceOptionsProvider, Order: 100
[info]: OmniSharp.WorkspaceInitializer
        Invoking Workspace Options Provider: OmniSharp.Roslyn.CSharp.Services.ImplementTypeWorkspaceOptionsProvider, Order: 110
[info]: OmniSharp.WorkspaceInitializer
        Invoking Workspace Options Provider: OmniSharp.Roslyn.CSharp.Services.EditorConfigWorkspaceOptionsProvider, Order: 200
[info]: OmniSharp.WorkspaceInitializer
        Configuration finished.
[info]: OmniSharp.Stdio.Host
        Omnisharp server running using Stdio at location '/my/project/path/MyProject' on host 5913.
```

**Comment by atrauzzi** [comment]
Just restarted fresh and got this error from OmniSharp:

For what it's worth, the project file does exist at the path it's looking, so I think that's a false negative.

```
Starting OmniSharp server at 2020-04-04, 11:00:01 a.m.
    Target: /home/atrauzzi/Development/atrauzzi/MySolution/MyProject

OmniSharp server started.
    Path: /home/atrauzzi/.vscode/extensions/ms-dotnettools.csharp-1.21.16/.omnisharp/1.34.15/run
    PID: 27459

[info]: OmniSharp.Stdio.Host
        Starting OmniSharp on ubuntu 19.10 (x64)
[info]: OmniSharp.Services.DotNetCliService
        DotNetPath set to dotnet
[info]: OmniSharp.MSBuild.Discovery.MSBuildLocator
        Located 1 MSBuild instance(s)
            1: StandAlone 16.4 - "/home/atrauzzi/.vscode/extensions/ms-dotnettools.csharp-1.21.16/.omnisharp/1.34.15/omnisharp/.msbuild/Current/Bin"
[info]: OmniSharp.MSBuild.Discovery.MSBuildLocator
        MSBUILD_EXE_PATH environment variable set to '/home/atrauzzi/.vscode/extensions/ms-dotnettools.csharp-1.21.16/.omnisharp/1.34.15/omnisharp/.msbuild/Current/Bin/MSBuild.dll'
[info]: OmniSharp.MSBuild.Discovery.MSBuildLocator
        Registered MSBuild instance: StandAlone 16.4 - "/home/atrauzzi/.vscode/extensions/ms-dotnettools.csharp-1.21.16/.omnisharp/1.34.15/omnisharp/.msbuild/Current/Bin"
            CscToolExe = csc.exe
            MSBuildToolsPath = /home/atrauzzi/.vscode/extensions/ms-dotnettools.csharp-1.21.16/.omnisharp/1.34.15/omnisharp/.msbuild/Current/Bin
            CscToolPath = /home/atrauzzi/.vscode/extensions/ms-dotnettools.csharp-1.21.16/.omnisharp/1.34.15/omnisharp/.msbuild/Current/Bin/Roslyn
            BypassFrameworkInstallChecks = true
            MSBuildExtensionsPath = /home/atrauzzi/.vscode/extensions/ms-dotnettools.csharp-1.21.16/.omnisharp/1.34.15/omnisharp/.msbuild
[info]: OmniSharp.Cake.CakeProjectSystem
        Detecting Cake files in '/home/atrauzzi/Development/atrauzzi/MySolution/MyProject'.
[info]: OmniSharp.Cake.CakeProjectSystem
        Could not find any Cake files
[info]: OmniSharp.MSBuild.ProjectSystem
        No solution files found in '/home/atrauzzi/Development/atrauzzi/MySolution/MyProject'
[info]: OmniSharp.MSBuild.ProjectManager
        Queue project update for '/home/atrauzzi/Development/atrauzzi/MySolution/MyProject/MyProject.csproj'
[info]: OmniSharp.Script.ScriptProjectSystem
        Detecting CSX files in '/home/atrauzzi/Development/atrauzzi/MySolution/MyProject'.
[info]: OmniSharp.Script.ScriptProjectSystem
        Found 1 CSX files.
[info]: OmniSharp.Script.ScriptContextProvider
        Searching for compilation dependencies with the fallback framework of 'net461'.
[info]: OmniSharp.MSBuild.ProjectManager
        Loading project: /home/atrauzzi/Development/atrauzzi/MySolution/MyProject/MyProject.csproj
[info]: OmniSharp.Script.ScriptContextProvider
        Unable to find dependency context for CSX files. Will default to non-context usage (Desktop CLR scripts).
[info]: OmniSharp.MSBuild.ProjectManager
        Successfully loaded project file '/home/atrauzzi/Development/atrauzzi/MySolution/MyProject/MyProject.csproj'.
[info]: OmniSharp.MSBuild.ProjectManager
        Adding project '/home/atrauzzi/Development/atrauzzi/MySolution/MyProject/MyProject.csproj'
[fail]: OmniSharp.MSBuild.ProjectManager
        Failed to add project to workspace: '/home/atrauzzi/Development/atrauzzi/MySolution/MyProject/MyProject.csproj'
[fail]: OmniSharp.MSBuild.ProjectManager
        Could not locate project in workspace: /home/atrauzzi/Development/atrauzzi/MySolution/MyProject/MyProject.csproj
[info]: OmniSharp.Script.ScriptProjectSystem
        Added CSX project '/home/atrauzzi/Development/atrauzzi/MySolution/MyProject/csx/mucking.csx' to the workspace.
[info]: OmniSharp.WorkspaceInitializer
        Invoking Workspace Options Provider: OmniSharp.Roslyn.CSharp.Services.CSharpFormattingWorkspaceOptionsProvider, Order: 0
[info]: OmniSharp.WorkspaceInitializer
        Invoking Workspace Options Provider: OmniSharp.Roslyn.CSharp.Services.RenameWorkspaceOptionsProvider, Order: 100
[info]: OmniSharp.WorkspaceInitializer
        Invoking Workspace Options Provider: OmniSharp.Roslyn.CSharp.Services.ImplementTypeWorkspaceOptionsProvider, Order: 110
[info]: OmniSharp.WorkspaceInitializer
        Invoking Workspace Options Provider: OmniSharp.Roslyn.CSharp.Services.EditorConfigWorkspaceOptionsProvider, Order: 200
[info]: OmniSharp.WorkspaceInitializer
        Configuration finished.
[info]: OmniSharp.Stdio.Host
        Omnisharp server running using Stdio at location '/home/atrauzzi/Development/atrauzzi/MySolution/MyProject' on host 27054.
mono_refcount_increment: cannot increment a ref with value 0

=================================================================
Native Crash Reporting
=================================================================
Got a SIGABRT while executing native code. This usually indicates
a fatal error in the mono runtime or one of the native libraries
used by your application.
=================================================================

=================================================================
Native stacktrace:
=================================================================
0x4bc043 - /home/atrauzzi/.vscode/extensions/ms-dotnettools.csharp-1.21.16/.omnisharp/1.34.15/bin/mono : (null)
0x4bc3bc - /home/atrauzzi/.vscode/extensions/ms-dotnettools.csharp-1.21.16/.omnisharp/1.34.15/bin/mono : (null)
0x465ae0 - /home/atrauzzi/.vscode/extensions/ms-dotnettools.csharp-1.21.16/.omnisharp/1.34.15/bin/mono : (null)
0x4bb648 - /home/atrauzzi/.vscode/extensions/ms-dotnettools.csharp-1.21.16/.omnisharp/1.34.15/bin/mono : (null)
0x7fcebb8af540 - /lib/x86_64-linux-gnu/libpthread.so.0 : (null)
0x7fcebb3963eb - /lib/x86_64-linux-gnu/libc.so.6 : gsignal
0x7fcebb375899 - /lib/x86_64-linux-gnu/libc.so.6 : abort
0x7250c4 - /home/atrauzzi/.vscode/extensions/ms-dotnettools.csharp-1.21.16/.omnisharp/1.34.15/bin/mono : (null)
0x707c31 - /home/atrauzzi/.vscode/extensions/ms-dotnettools.csharp-1.21.16/.omnisharp/1.34.15/bin/mono : (null)
0x724fab - /home/atrauzzi/.vscode/extensions/ms-dotnettools.csharp-1.21.16/.omnisharp/1.34.15/bin/mono : (null)
0x7253d3 - /home/atrauzzi/.vscode/extensions/ms-dotnettools.csharp-1.21.16/.omnisharp/1.34.15/bin/mono : monoeg_g_logv
0x72548f - /home/atrauzzi/.vscode/extensions/ms-dotnettools.csharp-1.21.16/.omnisharp/1.34.15/bin/mono : monoeg_g_log
0x423c8b - /home/atrauzzi/.vscode/extensions/ms-dotnettools.csharp-1.21.16/.omnisharp/1.34.15/bin/mono : (null)
0x5eeac0 - /home/atrauzzi/.vscode/extensions/ms-dotnettools.csharp-1.21.16/.omnisharp/1.34.15/bin/mono : (null)
0x5f1223 - /home/atrauzzi/.vscode/extensions/ms-dotnettools.csharp-1.21.16/.omnisharp/1.34.15/bin/mono : (null)
0x5f1a54 - /home/atrauzzi/.vscode/extensions/ms-dotnettools.csharp-1.21.16/.omnisharp/1.34.15/bin/mono : (null)
0x5cf32a - /home/atrauzzi/.vscode/extensions/ms-dotnettools.csharp-1.21.16/.omnisharp/1.34.15/bin/mono : (null)
0x5ddc51 - /home/atrauzzi/.vscode/extensions/ms-dotnettools.csharp-1.21.16/.omnisharp/1.34.15/bin/mono : (null)
0x417476d0 - Unknown

=================================================================
Telemetry Dumper:
=================================================================
Pkilling 0x7fceb6f79700 from 0x7fce8f5e6700
Pkilling 0x7fceb69fe700 from 0x7fce8f5e6700
Pkilling 0x7fce8f1e4700 from 0x7fce8f5e6700
Pkilling 0x7fcebb34d780 from 0x7fce8f5e6700
Pkilling 0x7fce8d198700 from 0x7fce8f5e6700
Pkilling 0x7fce8cd96700 from 0x7fce8f5e6700
Pkilling 0x7fceb717e700 from 0x7fce8f5e6700
Pkilling 0x7fceba36f700 from 0x7fce8f5e6700
Pkilling 0x7fceb7789700 from 0x7fce8f5e6700
Pkilling 0x7fceb4407700 from 0x7fce8f5e6700
Pkilling 0x7fceb7383700 from 0x7fce8f5e6700
Pkilling 0x7fceb6bff700 from 0x7fce8f5e6700
Pkilling 0x7fce8f7e7700 from 0x7fce8f5e6700
Pkilling 0x7fce8f3e5700 from 0x7fce8f5e6700
Pkilling 0x7fce8efe3700 from 0x7fce8f5e6700
Pkilling 0x7fceb4077700 from 0x7fce8f5e6700
Could not exec mono-hang-watchdog, expected on path '/home/atrauzzi/.vscode/extensions/ms-dotnettools.csharp-1.21.16/.omnisharp/1.34.15/etc/../bin/mono-hang-watchdog' (errno 2)
Pkilling 0x7fce8d399700 from 0x7fce8f5e6700
Pkilling 0x7fce8cf97700 from 0x7fce8f5e6700
Pkilling 0x7fce8cb95700 from 0x7fce8f5e6700
Pkilling 0x7fce8ebd3700 from 0x7fce8f5e6700
Pkilling 0x7fceb7588700 from 0x7fce8f5e6700
Entering thread summarizer pause from 0x7fce8f5e6700
Finished thread summarizer pause from 0x7fce8f5e6700.

Waiting for dumping threads to resume

=================================================================
	External Debugger Dump:
=================================================================
mono_gdb_render_native_backtraces not supported on this platform, unable to find gdb or lldb

=================================================================
Basic Fault Address Reporting
=================================================================
Memory around native instruction pointer (0x7fcebb3963eb):0x7fcebb3963db  d2 4c 89 ce bf 02 00 00 00 b8 0e 00 00 00 0f 05  .L..............
0x7fcebb3963eb  48 8b 84 24 08 01 00 00 64 48 33 04 25 28 00 00  H..$....dH3.%(..
0x7fcebb3963fb  00 75 26 44 89 c0 48 81 c4 18 01 00 00 c3 0f 1f  .u&D..H.........
0x7fcebb39640b  80 00 00 00 00 48 8b 15 59 3a 1a 00 f7 d8 41 b8  .....H..Y:....A.

=================================================================
Managed Stacktrace:
=================================================================
at <unknown> <0xffffffff>
at System.Reflection.Assembly:InternalGetAssemblyName <0x000af>
at System.Reflection.AssemblyName:GetAssemblyName <0x0008f>
at Microsoft.CodeAnalysis.MonoGlobalAssemblyCache:CreateAssemblyNameFromFile <0x00027>
at <GetAssemblyIdentitiesAndPaths>d__6:MoveNext <0x00167>
at Microsoft.CodeAnalysis.MonoGlobalAssemblyCache:ResolvePartialName <0x0022f>
at Microsoft.CodeAnalysis.Scripting.Hosting.GacFileResolver:Resolve <0x0005c>
at Microsoft.CodeAnalysis.Scripting.Hosting.RuntimeMetadataReferenceResolver:ResolveMissingAssembly <0x0009b>
at Microsoft.CodeAnalysis.Scripting.ScriptMetadataResolver:ResolveMissingAssembly <0x0003e>
at OmniSharp.Script.CachingScriptMetadataResolver:ResolveMissingAssembly <0x00089>
at Microsoft.CodeAnalysis.CommonReferenceManager`2:TryResolveMissingReference <0x000c0>
at Microsoft.CodeAnalysis.CommonReferenceManager`2:ResolveAndBindMissingAssemblies <0x0047b>
at Microsoft.CodeAnalysis.CommonReferenceManager`2:Bind <0x0028b>
at ReferenceManager:CreateAndSetSourceAssemblyFullBind <0x004d7>
at ReferenceManager:CreateSourceAssemblyForCompilation <0x0004b>
at Microsoft.CodeAnalysis.CSharp.CSharpCompilation:GetBoundReferenceManager <0x00043>
at Microsoft.CodeAnalysis.CSharp.CSharpCompilation:get_SourceAssembly <0x00027>
at Microsoft.CodeAnalysis.CSharp.CSharpCompilation:get_Assembly <0x00027>
at Microsoft.CodeAnalysis.CSharp.CSharpCompilation:get_SourceModule <0x0002f>
at Microsoft.CodeAnalysis.CSharp.CSharpCompilation:get_CommonSourceModule <0x00027>
at Microsoft.CodeAnalysis.Compilation:get_SourceModule <0x0002b>
at Microsoft.CodeAnalysis.Compilation:CommonBindScriptClass <0x000af>
at Microsoft.CodeAnalysis.CSharp.CSharpCompilation:BindScriptClass <0x00027>
at System.Lazy`1:ViaFactory <0x00082>
at System.Lazy`1:ExecutionAndPublication <0x000d3>
at System.Lazy`1:CreateValue <0x000e7>
at System.Lazy`1:get_Value <0x00033>
at Microsoft.CodeAnalysis.CSharp.CSharpCompilation:get_ScriptClass <0x00033>
at Microsoft.CodeAnalysis.CSharp.SyntaxTreeSemanticModel:GetDeclaredTypeMemberContainer <0x00107>
at Microsoft.CodeAnalysis.CSharp.SyntaxTreeSemanticModel:GetDeclaredSymbol <0x0013b>
at Microsoft.CodeAnalysis.CSharp.CSharpExtensions:GetDeclaredSymbol <0x00073>
at OmniSharp.Roslyn.CSharp.Services.Structure.CodeStructureService:CreateCodeElement <0x0006f>
at <CreateCodeElements>d__5:MoveNext <0x005d7>
at <GetCodeElementsAsync>d__4:MoveNext <0x006a5>
at MoveNextRunner:InvokeMoveNext <0x00092>
at System.Threading.ExecutionContext:RunInternal <0x001c6>
at System.Threading.ExecutionContext:Run <0x00043>
at MoveNextRunner:Run <0x000bb>
at System.Threading.Tasks.AwaitTaskContinuation:RunOrScheduleAction <0x0007e>
at System.Threading.Tasks.Task:FinishContinuations <0x00163>
at System.Threading.Tasks.Task:FinishStageThree <0x00083>
at System.Threading.Tasks.Task`1:TrySetResult <0x000ef>
at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1:SetResult <0x000e3>
at System.Runtime.CompilerServices.AsyncValueTaskMethodBuilder`1:SetResult <0x00073>
at <GetTextAsync>d__31:MoveNext <0x0029f>
at MoveNextRunner:InvokeMoveNext <0x00092>
at System.Threading.ExecutionContext:RunInternal <0x001c6>
at System.Threading.ExecutionContext:Run <0x00043>
at MoveNextRunner:Run <0x000bb>
at System.Threading.Tasks.AwaitTaskContinuation:RunOrScheduleAction <0x0007e>
at System.Threading.Tasks.Task:FinishContinuations <0x00163>
at System.Threading.Tasks.Task:FinishStageThree <0x00083>
at System.Threading.Tasks.Task`1:TrySetResult <0x000ef>
at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1:SetResult <0x000e3>
at <GetTextAndVersionAsync>d__39:MoveNext <0x0028b>
at MoveNextRunner:InvokeMoveNext <0x00092>
at System.Threading.ExecutionContext:RunInternal <0x001c6>
at System.Threading.ExecutionContext:Run <0x00043>
at MoveNextRunner:Run <0x000bb>
at System.Threading.Tasks.AwaitTaskContinuation:RunOrScheduleAction <0x0007e>
at System.Threading.Tasks.Task:FinishContinuations <0x00163>
at System.Threading.Tasks.Task:FinishStageThree <0x00083>
at System.Threading.Tasks.Task`1:TrySetResult <0x000ef>
at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1:SetResult <0x000e3>
at <GetValueAsync>d__15:MoveNext <0x00c8b>
at MoveNextRunner:InvokeMoveNext <0x00092>
at System.Threading.ExecutionContext:RunInternal <0x001c6>
at System.Threading.ExecutionContext:Run <0x00043>
at MoveNextRunner:Run <0x000bb>
at System.Threading.Tasks.AwaitTaskContinuation:RunOrScheduleAction <0x0007e>
at System.Threading.Tasks.Task:FinishContinuations <0x00163>
at System.Threading.Tasks.Task:FinishStageThree <0x00083>
at System.Threading.Tasks.Task`1:TrySetResult <0x000e3>
at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1:SetResult <0x000d3>
at System.Runtime.CompilerServices.AsyncValueTaskMethodBuilder`1:SetResult <0x0003b>
at <DisposableWaitAsync>d__1:MoveNext <0x00353>
at MoveNextRunner:InvokeMoveNext <0x00092>
at System.Threading.ExecutionContext:RunInternal <0x001c6>
at System.Threading.ExecutionContext:Run <0x00043>
at MoveNextRunner:Run <0x000bb>
at System.Threading.Tasks.AwaitTaskContinuation:RunOrScheduleAction <0x0007e>
at System.Threading.Tasks.Task:FinishContinuations <0x00163>
at System.Threading.Tasks.Task:FinishStageThree <0x00083>
at System.Threading.Tasks.Task`1:TrySetResult <0x000b7>
at TaskNode:System.Threading.IThreadPoolWorkItem.ExecuteWorkItem <0x0002b>
at System.Threading.ThreadPoolWorkQueue:Dispatch <0x002ca>
at System.Threading._ThreadPoolWaitCallback:PerformWaitCallback <0x0005b>
at <Module>:runtime_invoke_bool <0x0007e>
=================================================================
/home/atrauzzi/.vscode/extensions/ms-dotnettools.csharp-1.21.16/.omnisharp/1.34.15/run: line 28: 27463 Aborted                 (core dumped) "${mono_cmd}" "${omnisharp_cmd}" "$@"
```

**Comment by atrauzzi** [comment]
Possibly related? https://github.com/OmniSharp/omnisharp-vscode/issues/3701

**Comment by cartermp** [comment]
I don't thyink #3701 is related. But I do have a specific repro:

1. Clone Tye: https://github.com/dotnet/tye
2. Run the build script.
3. `code .` in the root directory.
4. Select `tye.sln` as the active solution.

Once everything is finished loading (it takes a little while), there is no IntelliSense, tooltips, etc.

**Comment by rjurney** [comment]
@cartermp I have the same issue on a modern VS Code and tye no longer installs because of a download error: 

```
./build.sh

dotnet-install: Downloading link: https://dotnetcli.azureedge.net/dotnet/Sdk/5.0.100-rc.1.20452.10/dotnet-sdk-5.0.100-rc.1.20452.10-osx-x64.tar.gz
dotnet-install: Extracting zip from https://dotnetcli.azureedge.net/dotnet/Sdk/5.0.100-rc.1.20452.10/dotnet-sdk-5.0.100-rc.1.20452.10-osx-x64.tar.gz
cp: /var/folders/0b/74l_65015_5fcbmbdz1w2xl40000gn/T//dotnet.3wVvdhj6p/\033[1;35;40m\033[K/var/folders/0b/74l_65015_5fcbmbdz1w2xl40000gn/T//dotnet.3wVvdhj6p/host/fxr/5.0.0-rc.1.20451.14/\033[m\033[K: No such file or directory
Failed to install dotnet SDK from public location (exit code '1').
Failed to install dotnet SDK from public location (exit code '1').
```

**Comment by JoeRobich** [comment]
closing due to a lack of recent reports. if this is still a problem, please open a new issue.
